extensions/amp-inputmask/README.md

```diff
@@ -124,6 +127,29 @@ The characters "+", "1", " ", "(", ")" and "-" will be automatically added as th
 <input type="tel" mask="+1_(000)_000-0000" placeholder="+1 (555) 555-5555">
 ```
 
+The following named masks are supported:
+
+<table>
+<tr>
+<th width="30%"><code>mask</code><br></th>
+<th>Description</th>
+</tr>
+<tr>
+<td><code>credit</code></td>
```

Comment:
Is there a reason we are going with `credit` instead of `payment-card` which is more generic?

Reply:
I chose credit arbitrarily. I'm glad to bikeshed on the name, I agree credit could be confusing for a dev if they think it won't validate debit cards for example. We could go from as generic as `payment-card` to `iec7812` for the numbering standard. https://en.wikipedia.org/wiki/ISO/IEC_7812